fly/commands/internal/setpipelinehelpers/atc_config.go

```diff
@@ -56,6 +58,18 @@ func (atcConfig ATCConfig) Set(yamlTemplateWithParams templatehelpers.YamlTempla
 		return err
 	}
 
+	configWarnings, _ := configvalidate.Validate(newConfig)
+	for _, w := range configWarnings {
+		atcConfig.CommandWarnings = append(atcConfig.CommandWarnings, concourse.ConfigWarning{
+			Type:    w.Type,
+			Message: w.Message,
+		})
+	}
+
+	if len(atcConfig.CommandWarnings) > 0 {
+		displayhelpers.ShowWarnings(atcConfig.CommandWarnings)
+	}
+
 	diffExists := diff(existingConfig, newConfig)
 
```

Comment:
Love this client side validation, but since we print it before the diff, it could be off screen when the pipeline yaml is long. What if we print it after we print out the diff?